tbx/static_src/sass/config/_variables.scss

```diff
@@ -177,10 +182,12 @@ $grid-gutters-x-large: 120px;
 
 // Vertical spacing
 $spacer-mini: 15px;
+$spacer-mini-plus: 20px;
 $spacer-small: ($spacer-mini * 2); // 30px
 $spacer-medium: ($spacer-mini * 4); // 60px
 $spacer-large: ($spacer-mini * 8); // 120px
 $spacer: 160px;
+$spacer-half: ($spacer / 2); // 80px
```

Comment:
Q: Should we use `math.div` here? (or `* 0.5`)

Reply:
actually I'm surprised the tooling didn't complain about this.